quic/s2n-quic-core/src/transport/parameters/mod.rs

```diff
@@ -1063,6 +1070,119 @@ impl From<connection::id::LocalId> for InitialSourceConnectionId {
 connection_id_parameter!(RetrySourceConnectionId, LocalId, 0x10);
 optional_transport_parameter!(RetrySourceConnectionId);
 
+/// Used by the client to indicate which versions of s2n-quic-dc it supports
+/// and by the server to indicate which version it is using
+#[derive(Clone, Copy, Debug, Default, PartialEq, PartialOrd, Eq, Ord)]
+pub struct DcSupportedVersions([VarInt; 4]);
```

Comment:
Can we put an explicit `len: u8` on here? I'd prefer not to have to iterate through the list every time to find the first `0` entry.